itou/templates/static/maintenance.html

```diff
@@ -0,0 +1,16 @@
+{% extends "layout/base.html" %}
+{% load components %}
+
+{% block title %}Maintenance {{ block.super }}{% endblock %}
+
+
+{% block content %}
+    <section class="s-section">
+        <div class="s-section__container container">
+            <h1>Maintenance en cours</h1>
+            <p>
+                Le service est actuellement en Maintenance, nous faisont tout notre possible pour rétablir le service au plus vite.
```

Comment:
```suggestion
                Le service est actuellement en maintenance, nous faisons tout notre possible pour rétablir le service au plus vite.
```
A voir pour la formulaire de la seconde partie car actuellement quand je la lit ça fait "C'est cassé, on est en train de réparer" sauf qu'en fait c'est un truc prévu et maîtrisé :thinking:.